index.html

```diff
@@ -0,0 +1,261 @@
+        </div>
+      </div>
+    </section>
+    <!-- About Section -->
+    <section class="about-page" id="about-page">
+      <div class="about-container">
+        <div class="about-content">
+          <h2 class="about-title">About Myself</h2>
+          <p class="about-para">
+            Hello I'm a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don't
+            hestiate to contact me.
+          </p>
+          <h3 class="about-contact">LET'S CONNECT</h3>
+          <ul class="social-icons">
+            <li class="icons">
+              <a href="#"><img src="images/Vector (15).png" alt="Twitter" /></a>
+            </li>
+            <li class="icons">
+              <a href="#"
+                ><img src="images/Linkedin icon.png" alt="LinkedIn"
+              /></a>
+            </li>
+            <li class="icons">
+              <a href="#"><img src="images/Vector (11).png" alt="Medium" /></a>
+            </li>
+            <li class="icons">
+              <a href="#"><img src="images/Vector (12).png" alt="GitHub" /></a>
+            </li>
+            <li class="icons">
+              <a href="#"
+                ><img src="images/Vector (13).png" alt="AngelList"
+              /></a>
+            </li>
+          </ul>
+          <button class="about-btn">Get my resume</button>
+        </div>
+        <div class="about-lang">
+          <div class="languages">
+            <div class="lang-title">
+              <h3>Languages</h3>
+              <img  src="images/Union.svg" alt="arrow" />
+            </div>
+
+            <ul>
+              <li class="langs">
+                <img src="images/Ellipse 1.png" alt="JavaScript-Logo" />
+                <h4 class="langs-text">JavaSript</h4>
+              </li>
+            </ul>
+            
+            <ul>
+              <li class="langs">
+                <img src="images/Ellipse 2.png" alt="HTML5-Logo" />
+                <h4 class="langs-text">HTML</h4>
+              </li>
+            </ul>
+            
+            <ul>
+              <li class="langs">
+                <img src="images/Ellipse 3.png" alt="CSS3-Logo" />
+                <h4 class="langs-text">CSS</h4>
+              </li>
+            </ul>
+          </div>
+          <div>
+            <ul>
+              <div class="framework">
+              <li><h4 class="frame-title">Frameworks</h4></li>
+              <li><img src="images/Union2.svg" alt="arrow" /></li>   
+              </div>
+              <hr class="new1" />
+            </ul>
+          </div>
+          <div>
+          <ul>
+            <div class="skill">
+             <li><h4 class="skill-title">Skills</h4></li> 
+             <li><img  src="images/Union2.svg" alt="arrow" /></li>
+            </div>
+           <hr class="new1" />
+          </div>
+          </ul>
+        </div>
```

Comment:
- [ ] You are taking multiple `ul`s here in this section which is against semantics and best practices. what the previous reviewer meant was that there will be one main `ui` which will include `languages, skills and frameworks`, each of these will be a `li` for that main `ul`.
then each `li` will include a `ul` for its `li`s. 
I am going to show you how to structure this so that so will fix it easily. kindly structure this section as in this following structure. ignore the classes names, you just need to follow the html structure.
```
<ul>
                        <li class="skills-list">
                            <div class="skill-list">
                                <h2>Languages</h2>
                                <img src="./images/arrow1.svg" alt="" class="collapsible">
                            </div>
                            <ul class="content skills-list">
                                <li>JavaScript</li>
                                <li>Ruby</li>
                                <li>HTML</li>
                                <li>CSS</li>
                            </ul>
                        </li>
                        <li class="skills-list">
                            <div class="skill-list">
                                <h2>Frameworks</h2>
                                <img src="./images/arrow1.svg" alt="" class="collapsible">
                            </div>
                            <ul class="content skills-list">
                                <li>React & Redux</li>
                                <li>Ruby on Rails</li>
                                <li>Tailwind CSS</li>
                                <li>Bootstrap</li>
                                <li>Jest</li>
                                <li>RSpec</li>
                            </ul>
                        </li>
                        <li class="skills-list">
                            <div class="skill-list">
                                <h2>Skills</h2>
                                <img src="./images/arrow1.svg" alt="" class="collapsible">
                            </div>
                            <ul class="content skills-list">
                                <li>Postgresql</li>
                                <li>Github</li>
                                <li>Git</li>
                                <li>Heroku, Netlify</li>
                                <li>Code Reviewer</li>
                                <li>Pair-Programming</li>
                            </ul>
                        </li>
                    </ul>
  ```

Reply:
I have changed the script exactly the way you mentioned. Please follw the script below:
<ul>
            <li class="languages">
                <div class="lang-title">
                  <h2>Languages</h2>
                  <img  src="images/Union.svg" alt="arrow" />
                </div>
                <ul>
                  <li class="langs">
                    <img src="images/Ellipse 1.png" alt="JavaScript-Logo" />
                    <h4 class="langs-text">JavaSript</h4>
                  </li>
                  <li class="langs">
                    <img src="images/Ellipse 2.png" alt="HTML5-Logo" />
                    <h4 class="langs-text">HTML</h4>
                  </li>
                  <li class="langs">
                    <img src="images/Ellipse 3.png" alt="CSS3-Logo" />
                    <h4 class="langs-text">CSS</h4>
                  </li>
                </ul>
            </li>
            <li>
                  <div class="framework">
                    <h2 class="frame-title">Frameworks</h2>
                    <img src="images/Union.svg" alt="arrow" />   
                  </div>
                  <ul class="content skills-list">
                    <li>React & Redux</li>
                    <li>Ruby on Rails</li>
                    <li>Tailwind CSS</li>
                    <li>Bootstrap</li>
                    <li>Jest</li>
                    <li>RSpec</li>
                  </ul>
            </li>
            <li>
                  <div class="skill">
                    <h2 class="skill-title">Skills</h2> 
                    <img  src="images/Union.svg" alt="arrow" />
                  </div>
                  <ul class="content skills-list">
                    <li>Postgresql</li>
                    <li>Github</li>
                    <li>Git</li>
                    <li>Heroku, Netlify</li>
                    <li>Code Reviewer</li>
                    <li>Pair-Programming</li>
                  </ul>
            </li>
          </ul>